frontend/src/pages/projects/screens/detail/notebooks/NotebookImageDisplayName.tsx

```diff
@@ -116,8 +119,27 @@ export const NotebookImageDisplayName = ({
   if (notebookImage.imageAvailability === NotebookImageAvailability.ENABLED) {
     return (
       <>
-        <HelperText>
+        <HelperText style={{ display: 'flex', alignItems: 'center', gap: '8px' }}>
           <HelperTextItem>{notebookImage.imageDisplayName}</HelperTextItem>
+          {isProjectScopedAvailable && isImageStreamProjectScoped && (
+            <HelperTextItem>
+              <Label
+                isCompact
+                variant="outline"
+                color="blue"
+                data-testid="project-scoped-label"
+                icon={
+                  <img
+                    style={{ height: 20 }}
```

Comment:
can we use '20px' for consistency?

Reply:
Thanks, I should have made these changes when you left review comments in Pushpa's PR 😭